### ISSUE TITLE
ModuleNotFoundError: No module named 'tensorflow.contrib' 

### ISSUE THREAD

**Issue by piriboy** [dataplayer12/homesecurity#14]
Hello
I keep getting following error. 
Any help would be appreciated.
Thanks

jetson@chung-jetson:/media/jetson/EXTDR/homesecurity$ python3 jetsontx2/compile_ssd_mobilenet.py
2020-09-09 12:52:44.261984: I tensorflow/stream_executor/platform/default/dso_loader.cc:48] Successfully opened dynamic library libcudart.so.10.2
Traceback (most recent call last):
  File "jetsontx2/compile_ssd_mobilenet.py", line 1, in <module>
    import tensorflow.contrib.tensorrt as trt
ModuleNotFoundError: No module named 'tensorflow.contrib'

**Comment by dataplayer12** [comment]
Sorry for the late response. Are you using tf1.x? This project was built to support only tensor flow 1.x and tensor flow 2.x is not supported.

**Comment by piriboy** [comment]
> Sorry for the late response. Are you using tf1.x? This project was built to support only tensor flow 1.x and tensor flow 2.x is not supported.

I see. Thanks!

**Comment by dataplayer12** [comment]
Was your issue resolved?

**Comment by dataplayer12** [comment]
Closing because issue seems to have been resolved.

**Comment by piriboy** [comment]
Thank you for following up.
I installed tf 1.7 but have not had a chance to test.
Thanks again.
Hoon

Donghoon Chung, Ph.D.
Associate Professor
Center for Predictive Medicine
Department of Microbiology Immunology
School of Medicine, University of Louisville
(P) 502-852-5522
dhchun01@louisville.edu<mailto:dhchun01@louisville.edu>

On Sep 17, 2020, at 2:22 AM, Jaiyam Sharma <notifications@github.com<mailto:notifications@github.com>> wrote:

CAUTION: This email originated from outside of our organization. Do not click links, open attachments, or respond unless you recognize the sender's email address and know the contents are safe.


Closing because issue seems to have been resolved.
